utils/supabase/supabase.ts

```
@@ -0,0 +1,55 @@
+import { createBrowserClient } from "@supabase/ssr"
+import { createServerClient } from "@supabase/ssr"
+import { cookies } from "next/headers"
+
+// Create a Supabase client for use in the browser
+export function createClient() {
+  return createBrowserClient(
+    process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)